src/datepicker/datepicker.component.ts

```diff
@@ -196,6 +212,7 @@ export class DatePicker implements OnDestroy, OnChanges, AfterViewChecked {
 	ngAfterViewChecked() {
 		if (!this.isFlatpickrLoaded()) {
 			this.flatpickrInstance = flatpickr(`#${this.id}`, this.flatpickrOptions);
+			this.flatpickrInstance._positionCalendar();
```

Comment:
Do we need this?

Reply:
The first time you open the calendar in the modal it is positioned slightly off:
![Screen Shot 2019-11-13 at 12 35 17 PM](https://user-images.githubusercontent.com/45505172/68788858-3fee9600-0612-11ea-8f7a-16b387ff2ae7.png)
Compare with:
![Screen Shot 2019-11-13 at 12 35 31 PM](https://user-images.githubusercontent.com/45505172/68788883-4a109480-0612-11ea-9234-e81dc1744e4e.png)
Using positionCalendar here fixes that.

Reply:
Interesting ... is this something we could hook into the `flatpickrBaseOptions.onOpen` callback?

Reply:
I can't seem to reproduce this anymore, might be some weird margins affecting the positioning before